docs/experiment/ppl/cmd/dedup.rst

```diff
@@ -0,0 +1,112 @@
+* keepempty: optional. if true, keep the document if the any field in the field-list has NULL value or field is MISSING. **Default:** false.
+* consecutive: optional. If set to true, removes only events with duplicate combinations of values that are consecutive. **Default:** false.
+* field-list: mandatory. The comma-delimited field list. At least one field is required.
+
+
+Example 1: Dedup by one field
+=============================
+
+The example show dedup the document with gender field.
+
+PPL query::
+
+    od> source=accounts | dedup gender | fields account_number, gender;
+    fetched rows / total rows = 2/2
+    +------------------+----------+
+    | account_number   | gender   |
+    |------------------+----------|
+    | 1                | M        |
+    | 13               | F        |
+    +------------------+----------+
+
+Example 2: Keep 2 duplicates documents
+======================================
+
+The example show dedup the document with gender field keep 2 duplication.
+
+PPL query::
+
+    od> source=accounts | dedup 2 gender | fields account_number, gender;
+    fetched rows / total rows = 3/3
+    +------------------+----------+
+    | account_number   | gender   |
+    |------------------+----------|
+    | 1                | M        |
+    | 6                | M        |
+    | 13               | F        |
+    +------------------+----------+
+
+Example 3: Ignore the empty field by default
+============================================
+
+The example show dedup the document by ignore the empty value field.
+
+PPL query::
+
+    od> source=accounts | dedup email | fields account_number, email;
+    fetched rows / total rows = 3/3
+    +------------------+-----------------------+
+    | account_number   | email                 |
+    |------------------+-----------------------|
+    | 1                | amberduke@pyrami.com  |
+    | 6                | hattiebond@netagy.com |
+    | 18               | daleadams@boink.com   |
+    +------------------+-----------------------+
+
+Example 4: Keep the empty field
+===============================
+
+The example show dedup the document by keep null value field.
+
+PPL query::
+
+    od> source=accounts | dedup email keepempty=true | fields account_number, email;
+    fetched rows / total rows = 4/4
+    +------------------+-----------------------+
+    | account_number   | email                 |
+    |------------------+-----------------------|
+    | 1                | amberduke@pyrami.com  |
+    | 6                | hattiebond@netagy.com |
+    | 13               | null                  |
+    | 18               | daleadams@boink.com   |
+    +------------------+-----------------------+
+
```

Comment:
What about removing example 3 or merging example 3 & 4 to one example by making 3 as the comparison case?

Reply:
goot point, done.